### ISSUE TITLE
set up redirects for urls that include a search term

### ISSUE THREAD

**Issue by jgravois** [Esri/esri.github.com#54]
http://esri.github.io/#Leaflet

needs to redirect to:

https://github.com/Esri?q=Leaflet

**Comment by jf990** [comment]
Also 

http://esri.github.io/?q=Leaflet

needs to redirect to:

https://github.com/Esri?q=Leaflet

**Comment by jgravois** [comment]
this works well. a little _too well_ in fact.

once we trap and avoid redirecting the [valid inline links](https://github.com/Esri/esri.github.com/blob/b734894dc6fe277c8d49e96de7ecf050886edf27/src/index.html#L23-L32) AL added yesterday, this'll be good to go.
```html
<a href="#webdevelopment" >Web Development</a>
<a href="#datamanagement" >Data Management</a>
<a href="#spatialanalysis" >Spatial Analysis</a>
<a href="#publishingsharing" >Publishing and Sharing</a>
```

**Comment by jgravois** [comment]
resolved by #60
